code/modules/mob/dead/observer/observer.dm

```diff
@@ -537,6 +538,19 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 	set hidden = 1
 	to_chat(src, "<span class='warning'>You are dead! You have no mind to store memory!</span>")
 
+
+/mob/dead/observer/verb/toggle_health_scan()

```

Comment:
I really don't think we need more verbs.
Maybe make it an option in the right-click menu, or a link that pops up when you examine them.